src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2

```diff
@@ -226,19 +118,20 @@ def start():
 {% endfor %}
 
         cols = st.columns(7)
-        submit_bt = cols[0].form_submit_button(label='Submit')
+        submit_bt = cols[0].form_submit_button(label='{{label}}', type='primary')
```

Comment:
add some placeholder for st_quill

Reply:
offline synced, will keep current behavior.